src/backend/nodes/outfuncs.c

```diff
@@ -2838,9 +2837,7 @@ _outPathTarget(StringInfo str, const PathTarget *node)
 	WRITE_FLOAT_FIELD(cost.per_tuple, "%.2f");
 	WRITE_INT_FIELD(width);
 }
-#endif /* COMPILING_BINARY_FUNCS */
 
-#ifndef COMPILING_BINARY_FUNCS
```

Comment:
Can you explain me why were these two functions exposed?

Reply:
```
#ifndef COMPILING_BINARY_FUNCS
...
#ifndef COMPILING_BINARY_FUNCS  <---- I removed this one
...
#endif
...
#endif
```